### ISSUE TITLE
about livereload

### ISSUE THREAD

**Issue by Brannua** [docsifyjs/docsify-cli#174]
Excuse me,

I only serve just one document using docsify listening on port 3000 currently

but the live-reload is not available unless i change the port, such as execute `docsify serve . -p 1234`

Why I can not use the defalut port 3000 ?

And how can I detect live-reload port is being used ?

Thanks~

**Comment by sy-records** [comment]
The default ports are 3000 and 35729.

```
  --port, -p             Listen port.                   [number] [default: 3000]
  --livereload-port, -P  livereload Listen port.       [number] [default: 35729]
```

I can't reproduce your question, what is your environment?

**Comment by Brannua** [comment]
> 

Thanks

![image](https://user-images.githubusercontent.com/38487617/159205825-2bf56ca4-be90-4680-9082-a77119e3eb60.png)

**Comment by Brannua** [comment]
> Why I can not use the defalut port 3000 ?
> 
> And how can I detect live-reload port is being used ?

1. Why I can not use the defalut port 3000 ?

2. And how can I detect live-reload port is being used ?

**Comment by sy-records** [comment]
If you do not specify a port, the default is 35729.

You can check the browser console for error messages

**Comment by Brannua** [comment]
如果我使用`docsify serve .`启动服务，则控制台如下，livereload 消失，这该如何定位问题？
![image](https://user-images.githubusercontent.com/38487617/159217890-864c2fc5-2840-4b83-bf1f-2879ddeda287.png)

**Comment by sy-records** [comment]
@Brannua 你可以加我微信我帮你看看什么问题 ob_start

**Comment by Brannua** [comment]
> @Brannua 你可以加我微信我帮你看看什么问题 ob_start

好嘞，谢谢大佬 ！

**Comment by sy-records** [comment]
Use `lsof` to view port listening information

```bash
lsof -i:35729
```
If localhost is not available, you can try `0.0.0.0`

If the pwa service is used, then it may not take effect.

**Comment by Brannua** [comment]
https://user-images.githubusercontent.com/38487617/159430351-183b1c06-ed6d-482f-ae9c-26761d90d8ba.mp4
